UPGRADE-3.1.md

```diff
@@ -18,6 +18,8 @@ Form
  * Support for data objects that implements both `Traversable` and `ArrayAccess`
    in `ResizeFormListener::preSubmit` method has been deprecated and will be
    removed in Symfony 4.0.
+ * A new "force_submit" option has been added to allow request handlers to
+   submit a form even if its name is not a key of the submitted data array.
```

Comment:
This is not mandatory. The option is just available for anyone who wants to use it, but is totally transparent for pure BC.

Final words ?